doc/plugin.txt

```diff
@@ -20,7 +20,10 @@ Manage project plugins
         --browserify ................... Compile plugin JS at build time using 
                                          browserify instead of runtime.
 
-    remove <pluginid>|<name> [...] ..... Remove plugins with the given IDs/name.
+        --force-copying-src ............ forces copying source files from the plugin even if the same
```

Comment:
I would just name this option --force. This needs to be documented here: https://github.com/apache/cordova-cli/blob/master/doc/readme.md#cordova-plugin-command